app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/fragments/DataCollectionFragment.kt

```diff
@@ -164,7 +172,7 @@ class DataCollectionFragment : Fragment() {
                     toolbar.title = dataCollectionViewModel.portalItemTitle.value
                 }
                 R.id.popupAttributeListFragment -> {
-                    toolbar.title = identifyResultViewModel.identifiedPopup.value?.title
+                    toolbar.title = dataCollectionViewModel.portalItemTitle.value
```

Comment:
this is the change for not displaying the popup title in the app bar. Instead we are displaying the web map title here.
The Popup title is in the bottomsheet now, added to fragment_popup_attribute_list.xml